src/lib/utils/notifications/index.ts

```diff
@@ -32,6 +32,9 @@ type SendNotificationProps = BaseSendNotificationProps &
       }
   );
 
+// Need permissions to read expo tokens and send notifications without sharing expo tokens to the public
+const prisma = new PrismaClient();
+
```

Comment:
Should use the `authorizedPrismaClient` which has now been introduced in a file with the same name. Makes it more clear when reading the code that it is a "dangerous" client, and that it uses the same instance everywhere.